pom.xml

```diff
@@ -25,7 +25,7 @@
 		<slf4j.version>1.7.21</slf4j.version>
 		<logback.version>1.1.7</logback.version>
 		<guava.version>19.0</guava.version>
-		<apache.http.components.version>4.4</apache.http.components.version>
+		<apache.http.components.version>4.5.2</apache.http.components.version>
```

Comment:
I am sorry but I can not find a version 4.5.3 in the maven repository: http://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient